nf_core/modules/lint/main_nf.py

```diff
@@ -257,9 +257,27 @@ def check_process_section(self, lines, fix_version, progress_bar):
         self.warned.append(("process_standard_label", "Process label unspecified", self.main_nf))
     for i, l in enumerate(lines):
         url = None
+        l = l.strip(" '\"")
         if _container_type(l) == "bioconda":
             bioconda_packages = [b for b in l.split() if "bioconda::" in b]
-        l = l.strip(" '\"")
+        if _container_type(l) == "bioconda" or _container_type(l) == "conda":
+            match = re.search(r"params\.enable_conda", l)
+            if match is None:
+                self.passed.append(
+                    (
+                        "deprecated_enable_conda",
+                        f"Found deprecated parameter 'params.enable_conda' in the conda definition",
+                        self.main_nf,
+                    )
+                )
+            else:
+                self.failed.append(
+                    (
+                        "deprecated_enable_conda",
+                        f"Deprecated parameter 'params.enable_conda' correctly not found in the conda definition",
```

Comment:
I think error messages are swapped, aren't they? This should be the error message.

Reply:
fixed! thank you for catching